### PR TITLE
- VolumeQuery itwinviewer update

### PR DESCRIPTION
**Pull request by dustinlebsock**
Updates the Volume Query sample to use the iTwinViewer.

### CODE REVIEW

File: src/frontend-samples/volume-query-sample/VolumeQueryApp.tsx [hunk 1]
```diff
@@ -1,163 +1,45 @@
...
+import { Viewer } from "@bentley/itwin-viewer-react";
+import { VolumeQueryWidgetProvider } from "./VolumeQueryWidget";
+import { IModelConnection } from "@bentley/imodeljs-frontend";
+import { IModelViewportControlOptions } from "@bentley/ui-framework";
+import { VolumeQueryApi } from "./VolumeQueryApi";
+
+const uiProviders = [new VolumeQueryWidgetProvider()];
+
+const VolumeQueryApp: FunctionComponent = () => {
+  const sampleIModelInfo = useSampleWidget("Use the controls below to change the view attributes.", [SampleIModels.RetailBuilding]);
```
Comment (williamkbentley) [inline src/frontend-samples/volume-query-sample/VolumeQueryApp.tsx:16]:
```suggestion
  const sampleIModelInfo = useSampleWidget("Use the controls below to query elements inside, overlapping, and outside of the clip.", [SampleIModels.RetailBuilding]);
```

... or similar

File: src/frontend-samples/volume-query-sample/VolumeQueryWidget.tsx [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+  };
+
+  /* Changing colors of elements that are going to be overridden */
+  const _onColorPick = (colorValue: ColorDef, position: SectionOfColoring) => {
+    const previousColors = classifiedElementsColors;
+    previousColors[position] = colorValue;
+    setClassifiedElementsColors(previousColors);
+  };
+
+  // Display drawing and sheet options in separate sections.
```
Comment (williamkbentley) [inline src/frontend-samples/volume-query-sample/VolumeQueryWidget.tsx:207]: Remove

**Review by williamkbentley — CHANGES_REQUESTED** [review]
Code looks good. Some incorrect comments

File: src/frontend-samples/volume-query-sample/VolumeQueryApp.tsx (hunk 1 above)
Comment (williamkbentley) [reply src/frontend-samples/volume-query-sample/VolumeQueryApp.tsx:16]: Use the controls below to query and color spatial elements in the iModel using a volume box.

Comment (dustinlebsock) [reply src/frontend-samples/volume-query-sample/VolumeQueryApp.tsx]: Fixed.

File: src/frontend-samples/volume-query-sample/VolumeQueryWidget.tsx (hunk 2 above)
Comment (dustinlebsock) [reply src/frontend-samples/volume-query-sample/VolumeQueryWidget.tsx]: removed.

**Review by williamkbentley — APPROVED** [review]

**Pull request merged**